packages/venia-concept/src/actions/cart/__tests__/asyncActions.spec.js

```diff
@@ -897,6 +897,32 @@ describe('toggleCart', () => {
         expect(dispatch).not.toHaveBeenCalled();
     });
 
+    test('toggleCart thunk opens the drawer and cart is empty', async () => {
```

Comment:
```suggestion
    test('its thunk opens the drawer when cart is empty', async () => {
```

These tests are inside a `describe('toggleCart` block so you don't have to repeat the name of the function each time 👍

Reply:
I've applied suggested changes.